static/scss/header.scss

```
@@ -0,0 +1,61 @@
+// sass-lint:disable mixins-before-declarations
+
+.hero-content {
+  position: relative;
+}
+
+.hero-title {
+  background: rgba(black, 0.83);
+  max-width: 580px;
+
+  @include media-breakpoint-down(sm) {
+    max-width: 255px;
+  }
+
+  h1 {
+    color: white;
+    font-weight: 600;
+    font-family: Montserrat;
+    letter-spacing: 0;
+    font-size: 48px;
+    line-height: 48px;
+    padding: 40px 25px 12px 40px;
+
+    @include media-breakpoint-down(sm) {
+      font-size: 18px;
+      line-height: 18px;
+      padding: 22px 22px 0 22px;
+    }
+  }
+
+  h2 {
+    color: $subtitle-color;
+    font-family: Montserrat;
+    font-weight: 600;
+    letter-spacing: 0;
+    font-size: 36px;
+    line-height: 44px;
+    padding: 0 49px 26px 40px;
+
+    @include media-breakpoint-down(sm) {
+      font-size: 16px;
+      padding: 0 7px 18px 22px;
+      line-height: 24px;
+    }
+  }
+}
```

This can be simplified in a bunch of ways:
- The outer padding should be set by the containing element, not the individual elements within
- The font family definition is redundant

I didn't thoroughly check that these styles work in all screen widths, so please feel free to alter these suggestions

```suggestion
.hero-title {
  background: rgba(black, 0.83);
  max-width: 580px;
  padding: 40px;

  @include media-breakpoint-down(sm) {
    max-width: 255px;
    padding: 20px;
  }

  h1 {
    color: white;
    font-weight: 600;
    font-size: 48px;
    line-height: 48px;
    margin: 0 0 15px 0;

    @include media-breakpoint-down(sm) {
      font-size: 18px;
      margin: 0 0 10px 0;
    }
  }

  h2 {
    color: $subtitle-color;
    font-weight: 600;
    font-size: 36px;
    line-height: 44px;
    margin: 0;

    @include media-breakpoint-down(sm) {
      font-size: 16px;
      line-height: 24px;
    }
  }
}
```